tests/test_lib_user.py

```diff
@@ -60,9 +61,8 @@ def test_00_create_user(self):
         
         user_str = "{0!s}".format(user)
         self.assertTrue(user_str == "<root.resolver1@realm1>", user_str)
-        # check proper unicode() and str() handling
-        self.assertIsInstance(str(user), bytes)
-        self.assertIsInstance(unicode(user), unicode)
+        # check proper unicode() / str() handling
+        self.assertIsInstance(six.text_type(user), six.text_type)
```

Comment:
Hmm, I think if we change this like that, the test isn't very interesting anymore if run in a Python 2 environment, because there, we want str() to return bytes and unicode() to return unicode :-/ I think it would be more sensible to have something like
```python
if six.PY2:
   # the two tests with str() / unicode() we had before
else:
    self.assertIsInstance(str(user), str)
```

Reply:
:+1: